README.md

```diff
@@ -1,2 +1,25 @@
-# who-we-are
-This repository presents this organization
+# Friends of PrestaShop
+
+This repository presents Friends of PrestaShop community, its missions, intents and publics.
+
+## Publics
```

Comment:
I don't get the meaning of this. :thinking:

Reply:
hmm, it's more like "people we accept in the community"

=> Concerned public instead?

Reply:
"People behind the project" or simply "People"?

Reply:
Go for People